lib/livebook/update_check.ex

```diff
@@ -143,9 +141,27 @@ defmodule Livebook.UpdateCheck do
     end
   end
 
-  defp newer?(version) do
+  defp new_version(release) do
     current_version = Application.spec(:livebook, :vsn) |> List.to_string()
-    stable?(version) and Version.compare(current_version, version) == :lt
+
+    with %{
+           "tag_name" => "v" <> version,
+           "published_at" => published_at,
+           "draft" => false
```

Comment:
Unless you revisited :D

```suggestion
           "draft" => false,
           "prerelease" => false
```

Reply:
We already check the version for prerelease, so we are good. :)

Reply:
Ah right, perfect!